src/lib/orm/entities/UserGameIntegrationsEntity.ts

```diff
@@ -0,0 +1,9 @@
+import { BaseEntity, Entity, JoinColumn, OneToOne } from 'typeorm';
+import { UserEntity } from './UserEntity';
+
+@Entity('user_game_integrations', { schema: 'public' })
```

Comment:
"integration" singular please. It's the standard we follow. Each row is a single integration for that user.